<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QSlider" name="Seek">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    background: #1ED760;  /* Set your desired background color for the groove */
    height: 6px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #FFFFFF, stop:1 #EDEDED);  /* Set your desired background color for the handle */
    width: 15px;
    height: 15px;
    margin-left: -5px;
    margin-right: -5px;
    border-radius: 7.5px;
    border: 1px solid #EDEDED; /* Border color for a more defined look */
    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3); /* Shadow effect */
}

QSlider::add-page:horizontal {
    background: #EDEDED;  /* Set your desired background color for the increased volume part */
    border-radius: 3px;
}

QSlider::sub-page:horizontal {
    background: #1ED760;  /* Set the same background color as the groove for the rest of the slider */
    border-radius: 3px;
    margin-right: -1px;  /* Adjust the margin to ensure the transition between colors is smooth */
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PlayPause">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:35px;
color:rgb(255,255,255);
background-color: rgb(24, 173, 76);
padding:0px
}
QPushButton:Hover
{
background-color:rgb(44,193,96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SeekBackward">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:35px;
color:rgb(255,255,255);
background-color: rgb(24, 173, 76);
padding:0px
}
QPushButton:Hover
{
background-color:rgb(44,193,96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SeekForward">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:35px;
color:rgb(255,255,255);
background-color: rgb(24, 173, 76);
padding:0px
}
QPushButton:Hover
{
background-color:rgb(44,193,96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="Volume">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>18</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::vertical {
    background: white;  /* Set the overall background color to white */
    width: 0px;        /* Set the width of the slider */
}

QSlider::handle:vertical {
    background: white;  /* Set the handle color to white */
    border: 1px solid #ccc;
    width: 0px;        /* Set the width of the handle */
}

QSlider::add-page:vertical {
    background: rgb(24, 173, 76);  /* Set the color below the handle to the specified RGB color */
}
</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>623</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mute">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:10px;
color:rgb(255,255,255);
background-color: rgb(24, 173, 76);
padding:0px
}
QPushButton:Hover
{
background-color:rgb(44,193,96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
